### PR TITLE
don't write extra GATEWAY= in /etc/sysconfig/network

### LINKED ISSUE #6508: v11(EL8): /etc/sysconfig/network is malformated
**Describe the bug**
On EL8, after you passed through configurator and reboot, all network cards configured previously doesn't have any IP addresses.

**To Reproduce**
1. Go through Configurator and configure several interfaces
2. Reboot

=> All interfaces configured previously lost network configuration

**Expected behavior**
Network configuration is kept between reboots.

**Additional context**
Issue is caused by a wrong configuration inside `/etc/sysconfig/network`:

```
[root@pfel8dev vagrant]# cat /etc/sysconfig/network
# Created by anaconda
NOZEROCONF=yes
GATEWAY=GATEWAY=192.168.121.1
```
Consequently, NetworkManager ignored `/etc/sysconfig/network-scripts` files for all interfaces.

### PR DESCRIPTION
**Pull request by nqb**
# Description
- Remove additionnal `GATEWAY` added in `/etc/sysconfig/network`
- Don't recompute gateway using new commands
- Write GATEWAY when necessary in `/etc/sysconfig/network-scripts`

# Impacts
Gateway and interfaces configuration on EL8

# Issue
fixes #6508
Related to d5742d3c794a460abab45c1bf8e4516cf1f065be and #6466 

# Delete branch after merge
YES

### CODE REVIEW

File: html/pfappserver/root/interface/interface_rhel.tt [hunk 1]
```diff
@@ -5,7 +5,9 @@ BOOTPROTO=static
 IPADDR=[% ipaddr %]
 NETMASK=[% netmask %]
 DEFROUTE=[% defroute %]
-[% gateway %]
+[% IF gateway -%]
+GATEWAY=[% gateway %]
+[% ELSE -%]
```
Comment (julsemaan) [inline html/pfappserver/root/interface/interface_rhel.tt]: Shouldn't that be `END` ?

Comment (nqb) [reply html/pfappserver/root/interface/interface_rhel.tt]: You are 100% right, I adjusted code.

**Comment by nqb** [comment]
If review is ok, I will run a pipeline and check if everything is ok inside EL8 VM. I already did it but I want to be sure.

**Comment by julsemaan** [comment]
code looks good, it can be merged whenever the pipeline passes

**Comment by nqb** [comment]
Pipeline in progress: https://gitlab.com/inverse-inc/packetfence/-/pipelines/354547226

**Comment by nqb** [comment]
EL8 job failed due to issues with `packetfence_perl` package (in place of `packetfence-perl`). I re-run the job.

**Comment by nqb** [comment]
I look at network config of EL8 VM after a reboot: gateway is correctly defined in `/etc/sysconfig/network` and in `ifcfg-ethX` of the default interface.

I'm merging this PR.

**Pull request merged**